tests/registry-deployment_gcp_pod_volume_test.yaml

```diff
@@ -0,0 +1,15 @@
+suite: Test-volume
```

Comment:
nitpick - the two other test files start with `registry_`, but this one does not. Is there a reason for this?

Reply:
+1

Reply:
Thanks @ianstanton for pointing that out !
I've updated the name of test file to match with other test files